core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/add/AddElementsFromKafka.java

```diff
@@ -160,7 +161,7 @@ public void setOptions(final Map<String, String> options) {
     @Override
     public ValidationResult validate() {
         final ValidationResult result = Operation.super.validate();
-        if (null != bootstrapServers && bootstrapServers.length < 0) {
+        if (null != bootstrapServers && bootstrapServers.length < 1) {
```

Comment:
```suggestion
        if (null != bootstrapServers != null && bootstrapServers.length < 1) {
```
As you've already made a change i think this might as well get changed too